dev/NavigationView/NavigationView.cpp

```diff
@@ -81,8 +81,12 @@ constexpr int s_itemNotFound{ -1 };
 
 static winrt::Size c_infSize{ std::numeric_limits<float>::infinity(), std::numeric_limits<float>::infinity() };
 
+bool isInitialLoading = false;
```

Comment:
>isInitialLoading [](start = 5, length = 16)

remove this

Reply:
Missed that, thank you for pointing that out!